### ISSUE TITLE
Notre Dame

### ISSUE THREAD

**Issue by danielalen** [gentlegiantJGC/ACExplorer#3]
![005abf780f796d1cef8efe41d97d9f6](https://user-images.githubusercontent.com/13874337/56485315-1b13c380-6506-11e9-9a1d-9b01781cb319.png)
How to find a model of Notre Dame from assassin's creed 5？

**Comment by gentlegiantJGC** [comment]
The Notre Dame is stored as hundreds (perhaps thousands) of individual 3D models scattered throughout the forge file. The DataBlock files specify how these models are pieced together in the world. The world is sliced up into a grid pattern and each DataBlock holds the data for one of these blocks. You will need to find each DataBlock (see below) used in that area and export each chunk individually to OBJ format and then load them all at the same time (which is what I did for that photo).

I want to add another plugin to make finding these chunks of data easier but that is the easiest option for now. Also note that file reading is not perfect so some models are skipped but it will get you what you see in the photo.

I would suggest updating to ACExplorer V1.0.1 as I have fixed a mesh export issue that causes a very minor issue with the Notre Dame
Here is a list of the names of the DataBlocks that cover the Notre Dame:
Cell09005_DataBlock
Cell09018_DataBlock
Cell04801_DataBlock
Cell07359_DataBlock
Cell07539_DataBlock
Cell07763_DataBlock
Cell07764_DataBlock
Cell07765_DataBlock
Cell07774_DataBlock
Cell07888_DataBlock
Cell07889_DataBlock
Cell07890_DataBlock
Cell07891_DataBlock
Cell07892_DataBlock
Cell08016_DataBlock
Cell08017_DataBlock
Cell08018_DataBlock
Cell08019_DataBlock
Cell08020_DataBlock
Cell08144_DataBlock
Cell08145_DataBlock
Cell08146_DataBlock
Cell08147_DataBlock
Cell08272_DataBlock
Cell08273_DataBlock
